pkgs/development/mobile/pebblesdk/pebble-libs.nix

```diff
@@ -0,0 +1,59 @@
+{ pkgs, python27Packages }:
+
+rec {
+  libpebble2 = python27Packages.buildPythonPackage rec {
+    name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.